web/locales/en/messages.ftl

```diff
@@ -1612,7 +1612,7 @@ partnerships-our-partners = Our Partners
 
 # FIRST POST SUBMISSION CTA
 first-cta-header-text = Thank you for donating your voice clips!
-first-cta-subtitle-text = Would you like to share some information about how you speak?
+first-cta-subtitle-text = Would you like to share some information about yourself? It helps developers build more inclusive technology
```

Comment:
Maybe "By telling us about yourself you will help develoeprs build more inclusive technology", (make it explicit who is doing the helping and how they are helping). But the original is ok too...

Reply:
I think that's better